src/Chainweb/Version.hs

```diff
@@ -670,7 +671,7 @@ enableModuleNameFix
     -> BlockHeight
     -> Bool
 enableModuleNameFix v bh = case v of
-  Mainnet01 -> forHeight 500_000 -- TODO
+  Mainnet01 -> forHeight 448501 -- ~ 2020-04-02T12:00:00Z
```

Comment:
This fix doesn't apply to testnet? The height for Mainnet is otherwise correct.

Reply:
oh, we should enable it after the kill switch, too. I'll add it.